### PR TITLE
BETA: Register mosaic.is-a.dev

### PR DESCRIPTION
**Pull request by mg0the0goat00**
Added `mosaic.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/goguardian.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "mg0the0goat00",
+           "email": "133016005+mg0the0goat00@users.noreply.github.com",
+           "discord": "1160256967504318576"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=goguardian.is-a.dev,8d43491af0a4db74e567"
```
Comment (Stef-00012) [inline domains/goguardian.json:9]: Vercel is not supported

**Review by Stef-00012 — CHANGES_REQUESTED** [review]

File: domains/goguard1an.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mg0the0goat00",
+        "email": "s.marley.gooch@redclay.k12.de.us"
+    },
+    "record": {
+        "URL": "https://aboutme-b6sk.onrender.com/"
```
Comment (Stef-00012) [inline domains/goguard1an.json:7]: I cannot reach your website
"Not Found"

**Review by Stef-00012 — CHANGES_REQUESTED** [review]

**Comment by mg0the0goat00** [comment]
im sorry i changed the link to be https://aboutmeeeeeeeeeee.onrender.com

File: domains/goguard1an.json (hunk 2 above)
Comment (mg0the0goat00) [reply domains/goguard1an.json:7]: changed it to aboutmeeeeeeeeeee.onrender.com im sorry

**Pull request closed without merging**